oat-sa/literallycanvas-core
# fix: remove vulnerabilities

Ticket: https://oat-sa.atlassian.net/browse/ADF-1382

## What's Changed

- Removed vulnerabilities

## How to test
- Clone the repo and run `yarn audit`

## How to test on env
- go to http://test-viktar.playground.kitchen.it.taocloud.org:41531/
- login as a guest
- click start button
- click tools dropdown on the bottom-left corner
- choose Scratchpad
- the canvas will appear so you can test it

![image](https://github.com/oat-sa/literallycanvas-core/assets/135347298/61b4b2e1-f6c8-4736-be99-fd9ecb8f9da5)
